roles/addons/diskless/readme.rst

```diff
@@ -240,21 +255,39 @@ Notes:
 
 Before closing, also remember to clean dnf cache into the image chroot to save space.
 
+.. code-block:: text
+
+  # rm -rf /var/tmp/diskless/workdir/space_image/mnt/var/cache/dnf/*
```

Comment:
Can't we consider a nicer command? Does `dnf clean all --installroot /var/tmp/diskless/workdir/space_image/` work?

Reply:
yes its works
Thanks